examples/new-state/src/lib.rs

```diff
@@ -208,7 +208,7 @@ mod tests {
             Some(expected_token_count)
         );
 
-        let mut a_set_iter = state_reloaded.another_struct.a_set.iter();
+        let mut a_set_iter = state_reloaded.another_struct.a_set.iter().expect("Failed to get iterator");
```

Comment:
In tests it is better to use `expect_report` since that will make `cargo concordium` report the error. This won't since panics are not reported, the program just terminates.